js/script.js

```diff
@@ -119,13 +136,14 @@ function draw() {
     // Draw scrolling platform
     drawPlatform();
     // Draw meme mirror
-    drawMirror();
+    //drawMirror();
```

Comment:
Is this temporarily commented out until the toggle works?

Reply:
Until I add the new heart mirror